docs/transpile/defaults_and_configuration_options.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Transpilation defaults and configuration options
+description: Default settings and configuration options including optimization and approximation degree
+
+---
+
+# Transpilation default settings and configuration options
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device, to optimize the circuit for execution on present day noisy quantum systems, or both. You can pass circuits to the `transpile()` method and use all default settings, or you can fine-tune the transpilation by using the available parameters. 
+
+## All available parameters:
+
+Following are all of the available parameters for the `transpile()` method.  All parameters except `circuits` are optional.  For full details, see the (Transpiler API documentation)[https://docs.quantum-computing.ibm.com/api/qiskit/transpiler#transpiler-api].
+
+Do we want this whole list?  If so, I'll add descriptions from (https://docs.quantum-computing.ibm.com/api/qiskit/compiler#compilation-routines)
```

Comment:
I think it's worth highlighting there that there are two classes of arguments to `transpile()`, those that describe the target of compilation, these are things like `backend` and  `target` (along with the legacy BackendV1 formats like `basis_gates`, `inst_map`). By default most users just use `backend` and provide a backend object, only if you have a more advanced use case do you specify any of the other arguments to describe a backend. The other options are things that purely influence how the transpiler does it's job, these are things like `optimization_level` and the `*_method` arguments which are more commonly used to try and get better, different, or specific output from the transpiler.

Reply:
Agree with Matt. Also I think it is important to specify that abstract circuits need to be transpiled because the backends have only a limit set of basis gates, and cannot execute arbitrary operations. I generally think of the minimum function of the transpiler is to respect connectivity (and introduce SWAP gates where needed) and translate to basis gates.

Reply:
I see - I tried a version where they're separated out.  Let me know if you like it that way, or one long set.  

Also, I hadn't added the parameter descriptions yet because I'm not sure if we're keeping them, or if we should just leave them all in the API.   I'll do it however people want it.